Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ProcessedFile/ByFile.tsx

```diff
@@ -176,10 +185,16 @@ const ByFile: Application.Types.iByComponent = (props) => {
                                 <legend className="w-auto" style={{ fontSize: 'large' }}>Actions:</legend>
                                 <form>
                                     <div className="form-group">
-                                        <button className="btn btn-primary" hidden={props.Roles.indexOf('Administrator') < 0}
-                                            onClick={(event) => { event.preventDefault(); reprocessAll(); }}>Reprocess All {data.length}</button>
+                                        <button className={`btn btn-primary${inEnterprise ? '' : ' disabled'}`} hidden={props.Roles.indexOf('Administrator') < 0}
+                                            onMouseEnter={() => setHover('Bulk')} onMouseLeave={() => setHover('None')} data-tooltip={"BulkReload"}
+                                            onClick={(event) => {
+                                                event.preventDefault();
+                                                if (inEnterprise) reprocessAll();
+                                            }}>Reprocess All {data.length}</button>
                                     </div>
-                                   
+                                    <ToolTip Show={hover === 'Bulk' && !inEnterprise} Position={'bottom'} Target={"BulkReload"}>
+                                        Bulk reprocessing only available in Enterprise Edition.
+                                    </ToolTip>
```

Comment:
Maybe we should add this to a sepperate file since we probably use that same content in multiple places.
something like `<EnterpriseRestrictionTooltip Task={'Bulk Reprocessing'} Target={'BulkReload'} Show={hover === 'Bulk'}/>` 
And somewhere else:
```
EnterpriseRestrictionTooltip  = (props: {}) => {
   const config = useAppSelector(ConfigSlice.XDAConfig);
   const inEnterprise = React.useMemo(() => (config.EditionStatus['Enterprise'] ?? false), [config.EditionStatus]);
   
   if (inEnterprise)
      return null
    return <ToolTip Show={props.Show} Position={'bottom'} Target={props.Target}>
    {props.Task} is only available in the Enterprise Edition.
     </ToolTip>
}
```

Reply:
This is a good idea, I implemented something similar to what you describe here.